mkosi/initrd/resources/mkosi-initrd.md

```diff
@@ -0,0 +1,59 @@
+% mkosi-initrd(1)
+%
+%
+
+# NAME
+
+mkosi-initrd — Build initrds or unified kernel images for the current system
+using mkosi
+
+# SYNOPSIS
+
+`mkosi-initrd [options…]`
+
+# DESCRIPTION
+
+`mkosi-initrd` is wrapper on top of `mkosi` to simplify the generation of
+initrds and Unified Kernel Images for the current running system.
+
+# OPTIONS
+
+`--kernel-version=`
+:   Kernel version where to look for the kernel modules to include. Defaults to
+    the kernel version of the running system (`uname -r`).
+
+`--format=`, `-t`
+:   Output format. One of `cpio` (CPIO archive), `uki` (a unified kernel image
+    with the image in the `.initrd` PE section) or `directory` (for generating
+    an image directly in a local directory).
+
+`--output=`, `-o`
+:   Name to use for the generated output image file or directory. Defaults
+    to `initrd`.
+
+    Note that this only specifies the output prefix, depending on the
+    specific output format and compression used, the full output name might be
+    `initrd.cpio.zst`
+
+`--output-dir=`, `-O`
+:   Path to a directory where to place all generated artifacts. If this is
+    not specified and the directory `mkosi.output/` exists in the local
+    directory, it is automatically used for this purpose.
```

Comment:
This should mention we use the current working directory if this is not specified and `mkosi.output` does not exist.

Reply:
In fact using `mkosi.output` was not working, fixed.